lib/kube/proxy/auth_test.go

```diff
@@ -252,11 +258,26 @@ current-context: foo
 				},
 			},
 			assertErr: require.NoError,
+		}, {
+			desc:               "kubernetes_service, bad kube creds",
+			serviceType:        KubeService,
+			kubeconfigPath:     kubeconfigPath,
+			impersonationCheck: alwaysSucceeds,
```

Comment:
The setup looks correct, why would this fail?

Reply:
This, aswell as the other cases that have a `wants:` section that isnt empty fails as the changes in auth.go returns early if there isnt a tlsConfig so kubeCreds is always nil in these tests.